modules/javafx.controls/src/test/java/test/javafx/scene/control/DatePickerTest.java

```diff
@@ -737,6 +737,36 @@ public void testFocusLost() {
         stageLoader.dispose();
     }
 
+    /**
+     * When DatePicker looses focus with misformatted text in the editor,
+     * checks that the value is not changed, and the text is reverted to the value
+     */
+    @Test
+    public void testFocusLostWithTypo() {
+        //datePicker.setEditable(true); // other test cases have it .is this necessary?
+
+        Button button = new Button();
+        StageLoader stageLoader = new StageLoader(new HBox(datePicker, button));
+
+        // initial value
+        datePicker.setValue(LocalDate.of(2015, 03, 25));
+        assertEquals("3/25/2015",datePicker.getEditor().getText());
+
+        // set misformatted text
+        //stageLoader.getStage().requestFocus(); // other test cases have it . is this necessary?
+        datePicker.requestFocus();
+        datePicker.getEditor().setText("11/24/20xx");
+
+        // loosing focus triggers cancelEdit() because the text cannot be parsed
+        button.requestFocus();
+
+        // check that value remains unchanged, and text is reverted
+        assertEquals(LocalDate.of(2015, 03, 25),datePicker.getValue());
+        assertEquals("3/25/2015",datePicker.getEditor().getText());
```

Comment:
Minor: A space after comma is missing

Reply:
ok, I have committed the changes in the test class